.github/workflows/test.yml

```diff
@@ -57,7 +57,15 @@ jobs:
 
         # https://twitter.com/jonhoo/status/1571290371124260865
       - name: Run unit tests
-        run: cargo nextest run --locked --features std --all-targets
+        # The `erc20-flash-mint-example` package enables the `reentrant` feature
+        # on `openzeppelin-stylus`, which combined with `--all-targets` seems to
+        # cause a linker issue "error: undefined symbol: storage_flush_cache".
+        #
+        # This is a known problem with workspace feature unification:
+        # https://github.com/rust-lang/cargo/issues/4463
+        #
+        # To deal with this, we simply ignore `erc20-flash-mint-example` from the compilation.
+        run: cargo nextest run --locked --features std --all-targets --workspace --exclude erc20-flash-mint-example
```

Comment:
I think we could omit the whole of _examples/*_ directory for this step, it would speed up the step's execution

Reply:
Agreed

Reply:
Simplified all of the scripts with this in mind, CI should be much faster now